.github/workflows/ci_build_android.yml

```diff
@@ -0,0 +1,48 @@
+name: Build Android library
+
+on:
+  workflow_call:
+    outputs:
+      artifact-dir-base:
+        description: The base name of artifact directory (without the target suffix)
+        value: ${{ jobs.build-android-libs.outputs.artifact-dir-base }}
+
+jobs:
+  build-android-libs:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 35
+    strategy:
+      matrix:
+        target: [
+          { rust: "aarch64-linux-android", ndk: "arm64-v8a" },
```

Comment:
I'd rather just use the Rust target triple here to have less code

Reply:
Since this mapping is not exactly obvious, I think it's good to have here in the CI. Especially considering that the rust triplets don't really have meaning within the Android build process, and we'd be naming the zip artifacts based on them. Following your suggestion, we would build a `build-android-libs-aarch64-linux-android.zip`, which would then contain a directory `arm64-v8a`, leaving the mapping to the person who has to deal with the artifact. 

Is this a must-have from your perspective?

Reply:
> Following your suggestion, we would build a build-android-libs-aarch64-linux-android.zip, which would then contain a directory arm64-v8a, leaving the mapping to the person who has to deal with the artifact.

That's ok I think because these archives are just for sharing the artifacts between github jobs. It is not an artifact to be used by a person. The main purpose of the target name is to create a unique name for these artifacts within the matrix jobs